INSTALL.md

```diff
@@ -41,7 +41,7 @@ conda create -n optimade python=3.7
 conda activate optimade
 
 # Install package and dependencies in editable mode (including "dev" requirements).
-pip install -e .[dev]
+pip install -e ".[dev]"
```

Comment:
```suggestion
pip install -e .[dev]
```
Why the need to add the extra quotation marks here?

Reply:
usually `.` explicitly defines the current path and based on you chose of terminal environment might try to substitute the value.

For example in the case of zsh:
```bash
$ pip install -e .[dev]
zsh: no matches found: .[dev]
```
The `.[dev]` is just an argument of for pip so it is represented as a string (and handled by a python code).  I know it is a `zsh` related issue but representing as string work in ALL environments.

More about this issue:
- https://stackoverflow.com/questions/30539798/zsh-no-matches-found-requestssecurity
- https://stackoverflow.com/questions/39565867/zshell-pip-cant-find-package-autobahnserialization

Reply:
Huh, didn't know about this, and have never seen it anywhere else.
I am a bit hesitant to add something general due to a specific terminal's behavior, (from your linked stackoverflow links, it also seems one could add a conditional to the startup script for the terminal to avoid globbing pip).
But since it doesn't change the behavior (according to you) in any other terminal, then it's fine, I guess.